config/config_helpers.go

```diff
@@ -168,7 +170,7 @@ func processMultipleInterpolationsInString(terragruntConfigString string, includ
 // Given a string value from a Terragrunt configuration, parse the string, resolve any calls to helper functions using
 // Resolve a single call to an interpolation function of the format ${some_function()} in a Terragrunt configuration
 func resolveTerragruntInterpolation(str string, include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (interface{}, error) {
-	matches := HELPER_FUNCTION_SYNTAX_REGEX.FindStringSubmatch(str)
+	matches := INTERPOLATION_SYNTAX_REGEX.FindStringSubmatch(str)
```

Comment:
The HELPER_FUNCTION_SYNTAX_REGEX was a generalized version of the `INTERPOLATION_SYNTAX_REGEX` -- presumably with capture groups.  I couldn't think of a reason why I wouldn't just use `INTERPOLATION_SYNTAX_REGEX` after the capture groups were properly set up.